tests/component/views/create-dataset/CreateDataset.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import { CreateDataset } from '../../../../src/views/create-dataset/CreateDataset'
+
+describe('Form component', () => {
```

Comment:
Please, use testing library whenever is possible and try avoid using data-cy

Reply:
Understood! Removed the unnecessary attributes